server/views/case-summary-record-order.njk

```diff
@@ -182,6 +182,22 @@
                 </div>
             </div>
 
+            {% if currentOrder.licenceConditions | length %}
+                {% if custodialOrder.custodialTypeCode === "B" %}
+                    <h2 class="govuk-heading-m govuk-!-margin-top-8">Licence conditions</h2>
```

Comment:
You could inline this **if** to reduce a little duplication

`<h2 class="govuk-heading-m govuk-!-margin-top-8">{{"Licence conditions" if custodialOrder.custodialTypeCode === "B" else "Post-sentence supervision requirements"}}</h2>`

Reply:
🙆‍♀️